templates/etc/sysconfig/puppetserver.epp

```diff
@@ -25,6 +25,10 @@ JAVA_BIN="<%= $pupmod::master::sysconfig::java_bin %>"
 # Modify this if you'd like to change the memory allocation, enable JMX, etc
 JAVA_ARGS="-Xms<%= $java_start_memory %> -Xmx<%= $java_max_memory %> -Djava.io.tmpdir=<%= $pupmod::master::sysconfig::_java_temp_dir %> <%= $extra_java_args %>"
 
+<% if $pupmod::master::sysconfig::_use_jruby9k { -%>
```

Comment:
_(This is a general observation, not a suggestion for this issue)_

An alternate approach to manage this setting, using [puppetlabs-inifile](https://github.com/simp/puppetlabs-inifile) (from [petems-hiera_vault](https://github.com/petems/petems-hiera_vault/blame/8b54fbe735d6bb9862c2f2eca6853e84b3518947/README.md#L31-L40)):

```puppet
ini_setting { "Change jruby to 9k":
  ensure            => present,
  setting           => 'JRUBY_JAR',
  path              => "/etc/sysconfig/puppetserver",
  key_val_separator => '=',
  section           => '',
  value             => '"/opt/puppetlabs/server/apps/puppetserver/jruby-9k.jar"',
  show_diff         => true,
  notify            => Service['puppetserver']
}
```